pkg/webui/components/radio-button/radio.js

```diff
@@ -107,20 +108,22 @@ class RadioButton extends React.PureComponent {
       radioProps.name = this.context.name
       radioProps.disabled = disabled || this.context.disabled
       radioProps.checked = value === this.context.value
+      radioProps.id = `${this.context.name}-${stringToHash(value)}`
       groupCls = this.context.className
     } else {
       radioProps.name = name
       radioProps.disabled = disabled
       radioProps.checked = checked
       radioProps.value = value
+      radioProps.id = `${name}-${stringToHash(value)}`
     }
 
     const cls = classnames(className, style.wrapper, groupCls, {
       [style.disabled]: radioProps.disabled,
     })
 
     return (
-      <label className={cls}>
+      <label className={cls} htmlFor={radioProps.id}>
```

Comment:
this shouldnt be necessary as `<input />` is wrapped by `<label />`

Reply:
I know, but it also doesn't hurt and I wanted it to be consistent with the checkbox. Please unresolve this, if you disagree.